convert_image.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+###########################################################
+#
+# Author: noirmurir
+# Sending_Grounds Bash Scripting Challenge
+#
+# Shell frontend that processes the filenames and calls
+# the correct conversion utilities.
+# Optionally we can change the size of the resulting image,
+# place a colored border around an image
+# NetPBM utilities are used in this script to 
+# work with the images.
+#
+# Requirements:
+# - Catches signals
+# - Has a help/usage statement
+# - Uses named Exit codes
+# - Has arg parsing
+# - Posts it to this repo here
+# - and the script of course works
+# 
+###########################################################
+
+# Named exit codes
+EXIT_SUCCESS=0
+EXIT_INVALID_ARGS=1
+EXIT_SIGNAL_RECEIVED=2
+
+# Print help/usage
+usage() {
+cat<<EOF
+Usage: $0 [-S] [-s N] [-w N] [-c S] imagefile...
+
+Options:
+  -S          Change sharpness of a resulting image
+  -s N        Change size of a resulting image
+  -w N        Change width of a border of a resulting image
+  -c S        Change color of a border of a resulting image
+  -h          Show help/usage
+EOF
```

Comment:
This is really clean, I like it! The only thing I might suggest is that you add just a one-line or two on what the script does right above the usage.